shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -6,6 +6,7 @@ import collection.mutable.{HashMap, HashSet, Stack}
 import collection.immutable.LazyList
 import scala.collection.immutable
 import mlscript.codegen.{CodeGenError, Scope}
+import javax.xml.transform.Source
```

Comment:
Probably added by mistake.

Reply:
Oops, it seems auto-import malfunctioned.